CHANGES.rst

```diff
@@ -0,0 +1,4 @@
+0.3.1
+
+   - Bug fixes
+      - #670: Internal system error "dagster.check.CheckError: Invariant failed. Description: Should not be in context" raised when user throwing error during transform.
```

Comment:
This is a cryptic error message. When is it *supposed* to be raised?

Reply:
It really should never be raised. It is an internal framework error.

Reply:
"Key {key} should not be in context"?